WindowsServerDocs/virtualization/hyper-v/manage/Manage-Hyper-V-integration-services.md

```diff
@@ -307,4 +310,7 @@ For hosts earlier than Windows Server 2016 and Windows 10, you'll need to manual
   
 4.  After the installation finishes, all integration services are available for use.
 
-These steps can't be automated or done within a Windows PowerShell session for online virtual machines. You can apply them to offline VHDX images; [see this blog post](https://blogs.technet.microsoft.com/virtualization/2013/04/18/how-to-install-integration-services-when-the-virtual-machine-is-not-running/).
+> [!NOTE]
+> These steps **can't be automated** or done within a Windows PowerShell session for **online** virtual machines.
+> You can apply them to **offline** VHDX images; [see this article](https://docs.microsoft.com/en-us/virtualization/community/team-blog/2013/20130418-how-to-install-integration-services-when-the-virtual-machine-is-not-running).
```

Comment:
1. MS Docs recommend using language agnostic links (as long as the link does not break when removing the /en-us/ part of the link)
2. Links are normally labeled with the page title the link points to, unless the title is more or less part of the preceding description (and the title is long enough to become cluttering or less readable inside a note format blob sized text).
```suggestion
> You can apply them to **offline** VHDX images; see [How to install integration services when the virtual machine is not running](https://docs.microsoft.com/virtualization/community/team-blog/2013/20130418-how-to-install-integration-services-when-the-virtual-machine-is-not-running).
```